cluster-scope/base/operators.coreos.com/subscriptions/camel-k/subscription.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: camel-k
+  namespace: operators
```

Comment:
This namespace doesn't exist. I think you wanted to use `openshift-operators` instead. It's also generally better to enforce the namespace via `kustomization.yaml` (see this [Kiali example here](https://github.com/operate-first/apps/blob/master/cluster-scope/base/operators.coreos.com/subscriptions/kiali-operator/kustomization.yaml)).